Dockerfile

```diff
@@ -1,15 +1,20 @@
-FROM node:20.12.2
+FROM node:20.12.2-slim
+
+RUN apt-get update || : && apt-get install -y \
+    python3 \
+    build-essential \
+    jq
 
 ENV ENKETO_SRC_DIR=/srv/src/enketo
 WORKDIR ${ENKETO_SRC_DIR}
 
 COPY . ${ENKETO_SRC_DIR}
 
-# Install (including dev dependencies) and build
-RUN yarn install --frozen-lockfile
-
-# Remove dev dependencies after build
-RUN yarn install --production --frozen-lockfile --ignore-scripts
+# Install and build, first with dev dependencies, then with production only (yarn 1 has no prune)
+RUN yarn install --frozen-lockfile \
```

Comment:
This is weird, I agree. I decided to leave it because I don’t want to build yet another time. All of the necessary packages end up being built because of the dependencies between them.